src/artifact.cpp

```diff
@@ -544,6 +554,31 @@ static const std::array<std::string, 20> artifact_noun = { {
     translate_marker( "%s Graves" ), translate_marker( "%s Horrors" ), translate_marker( "%s Suffering" ), translate_marker( "%s Death" ), translate_marker( "%s Horror" )
 } };
 std::string artifact_name( const std::string &type );
+//Dreams for each charge req
+static const std::array<artifact_dream_datum, NUM_ACRS> artifact_dream_data = { {
+    {   {translate_marker("The %s is somehow vaguely dissatisfied even though it doesn't want anything. Seeing this is a bug!")},
+        {translate_marker("The %s is satisfied, as it should be because it has no standards. Seeing this is a bug!")},
+        100,  0
+    },{ {translate_marker("Your %s feels needy, like it wants to be held.")},
+        {translate_marker("You snuggle your %s closer.")},
+        50,  35
+    },{ {translate_marker("Your %s feels needy, like it wants to be touched.")},
+        {translate_marker("You press your %s against your skin.")},
+        50,  35
+    },{ {translate_marker("The %s is confused to find you dreaming while awake. Seeing this is a bug!")},
+        {translate_marker("Your %s sleeps soundly.")},
+        100, 33
+    },{ {translate_marker("Your %s longs for the glow.")},
+        {translate_marker("Your %s basks in the glow.")},
+        25,  75
+    },{ {translate_marker("You dream of angels' tears falling on your %s.")},
+        {translate_marker("You dream of playing in the rain with your %s.")},
+        50,  60
+    },{ {translate_marker("You dream that your %s is being buried alive.")},
```

Comment:
Can't say for others, but I'll try to bury my artifact if I had this dream. Maybe rephrase it somehow?

Reply:
Well, it's meant to be a little bit cryptic. That one kind of depends on how you look at it - it's intended to be "it is currently feeling claustrophobic and you should fix that", but if you take it at face value then yeah, you might think it means "you're pretty sure burying this is a good idea". My thinking is that "buried alive" carries an inherently negative connotation, implicitly communicating that it's not a good thing - and even if you don't get it, there are only so many ways to "bury" it before you'll try doing the opposite.

(Fun fact, I was originally going to make ACR_CAVE to require being below the surface, but then I realized that would interact rather poorly with ARTC_SOLAR.)

Honestly, I was more worried that some of them weren't cryptic *enough*... I was using the examples nexusmrsep suggested in #24153 (longs for the glow and angels' tears) as a baseline for level-of-crypticness, and trying to avoid just flat-out telling the player what it needs, but I couldn't think of anything particularly cryptic for ACR_EQUIP or ACR_SKIN.

Reply:
How does an inanimate object get buried alive?

Reply:
Either there's some sort of mind in there or you're just anthropomorphizing it, whichever you prefer to think.